controllers/btcrate-controller.js

```diff
@@ -0,0 +1,18 @@
+const btcRateService = require('../service/btcrate-service');
+
+class BTCRateController {
+
+    async btcRate(req, res, next){
+        try {
+            const userData = await btcRateService.btcRate();
+
+            console.log(userData);
```

Comment:
There is no need to `console.log`  in the business logic code. Either use a separate logger (like winston - these are external dependencies) or remove this line.